### ISSUE TITLE
Non-determinisic Lock Checker bug

### ISSUE THREAD

**Issue by smillst** [typetools/checker-framework#755]
On [Jenkins](http://tern.cs.washington.edu:8080/job/daikon-typecheck/jdk=JDK_8/884/) the Lock Checker issues the following erroneous errors, though [Travis](https://travis-ci.org/typetools/daikon-typecheck-lock/jobs/133452975) does not.

````
./daikon/split/PptSplitter.java:363: warning: [cannot.dereference] cannot dereference expression eq_inv.ppt with refined type annotation @GuardedByBottom
                    + eq_inv.ppt
                            ^
./daikon/split/PptSplitter.java:363: warning: [cannot.dereference] cannot dereference expression eq_inv.ppt with refined type annotation @GuardedByBottom
                    + eq_inv.ppt
````
